tests/README.md

```diff
@@ -0,0 +1,5 @@
+# Tests
+
+For information about how to create and run tests, see the [Tests](https://github.ibm.com/GoldenEye/documentation/blob/master/tests.md) doc.
```

Comment:
update to same as https://github.com/terraform-ibm-modules/terraform-ibm-module-template/blob/main/tests/README.md